security/bouncycastle-fips/src/test/java/io/quarkus/ts/security/bouncycastle/fips/BouncyCastleFipsJsseIT.java

```diff
@@ -0,0 +1,36 @@
+package io.quarkus.ts.security.bouncycastle.fips;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.CoreMatchers.containsString;
+
+import java.nio.file.Paths;
+
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.Dependency;
+import io.quarkus.test.services.QuarkusApplication;
+
+@QuarkusScenario
+public class BouncyCastleFipsJsseIT {
+
+    private static final String PASSWORD = "password";
+
+    @QuarkusApplication(dependencies = {
+            @Dependency(groupId = "org.bouncycastle", artifactId = "bctls-fips", version = "1.0.12.2")
+    })
+    static RestService app = new RestService().withProperties("jsse.properties");
+
+    @Test
+    public void verifyBouncyCastleAndJsseProviders() {
+        given()
+                .keyStore(Paths.get("client-keystore.jks").toFile(), PASSWORD)
```

Comment:
1) How were these keystore files generated? I would prefer to either leave an instruction somewhere, or eve to generate them in `@BeforeAll`.
2) `BouncyCastleFipsIT` doesn't use any of the store files. Is this intentional?

Reply:
A README.md was created for this module where this topic is explained.

> BouncyCastleFipsIT doesn't use any of the store files. Is this intentional?
Yes, is not need it